public/app/features/explore/RichHistory/RichHistoryQueriesTab.tsx

```diff
@@ -162,7 +162,7 @@ export function RichHistoryQueriesTab(props: Props) {
   return (
     <div className={styles.container}>
       <div className={styles.containerSlider}>
-        <div className={styles.slider}>
+        <div className={styles.slider} aria-label="slider">
```

Comment:
Nitpick: majority of our aria-labels start with a capital letter. Shouldn't we do the same there?